### ISSUE TITLE
Please include all makefiles or remove Makefile

### ISSUE THREAD

**Issue by rrthomas** [gjwgit/wajig#14]
`Makefile` contains the following:

```
# Required modules.

INC_BASE    = $(HOME)/.local/share/make
INC_CLEAN   = $(INC_BASE)/clean.mk
INC_PANDOC  = $(INC_BASE)/pandoc.mk
INC_GIT     = $(INC_BASE)/git.mk
INC_LATEX   = $(INC_BASE)/latex.mk
```

Note the path of `INC_BASE`. It doesn't seem therefore to be possible for the reader to know how this `Makefile` really works! Please could you either include the includes in git, or remove the `Makefile`?

**Comment by gjwgit** [comment]
Thanks Thomas. Probably best to remove. I'll do that shortly.

Regards,
Graham

**Comment by gjwgit** [comment]
Resolved in commit 652c1091a43851b89b2e8c4aac89b3cef903e9bd. Support files added. Thanks @rrthomas.
